backend/connector-integration/src/connectors/globalpay/transformers.rs

```diff
@@ -310,11 +311,9 @@ pub struct GlobalpayPaymentsRequest<T: PaymentMethodDataTypes> {
     pub country: common_enums::CountryAlpha2,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub capture_mode: Option<GlobalpayCaptureMode>,
```

Comment:
`skip_serializing_if` was removed for `initiator` field. Since this field is now always serialized (even when `None`), verify that the GlobalPay API accepts `null` values for optional fields.

If the API requires omitting the field when not present, keep the original attribute:

```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
    pub initiator: Option<Initiator>,
```